### ISSUE TITLE
Icon and desktop file for Linux systems

### ISSUE THREAD

**Issue by sanjayankur31** [pauldmccarthy/fsleyes#11]
Would it be possible to include a desktop file and an icon that can be used on Linux systems to list fsleyes in the "applications" list? Currently one must run it from a terminal, but since it's a GUI application, it'll be useful to have it listed in applications. Adding a desktop file and icon will also enable us to provide appstream data to make fsleyes available in the gnome-software etc. package installation tools.

- https://specifications.freedesktop.org/desktop-entry-spec/latest/apa.html
- https://www.freedesktop.org/wiki/Distributions/AppStream/

Neither of these are compulsory. They're "enhancements".

**Comment by sanjayankur31** [comment]
Yes, I don't know how python does desktop files either, but if these can be included in the source tar, we downstream packages can place them in the required locations. I can open PRs with desktop and appdata files if that would help?

**Comment by pauldmccarthy** [comment]
I've added a 48x48 application icon, and a file called `uk.ac.ox.fmrib.FSLeyes.desktop`, containing the following:

```
[Desktop Entry]
Type=Application
Name=FSLeyes
Comment=The FSL image viewer
TryExec=fsleyes
Exec=fsleyes -S %F
Icon=fsleyes
Categories=Science;DataVisualization;
```

Is there anything else needed? It would be great if you could open a PR with whatever else is necessary.

**Comment by pauldmccarthy** [comment]
Great - I'll hopefully have a new release out on pypi today or tomorrow ...

**Comment by sanjayankur31** [comment]
Thanks.  This github mirror is behind the main git repo (at ox.ac.uk). Could you update it please, otherwise my PR will not be based on the correct HEAD. I could pull master from there, but then those commits will become part of the PR too (and may cause conflicts?).

**Comment by pauldmccarthy** [comment]
I update the github mirrors whenever I do a new release ... unless I forget to :). Another thing that I should get CI to do for me.
